Packs/Darktrace/Integrations/Darktrace/Darktrace.yml

```diff
@@ -0,0 +1,491 @@
+category: Network Security
+commonfields:
+  id: Darktrace
+  version: -1
+configuration:
+- defaultvalue: https://example.net
+  display: Server URL (e.g. https://example.net)
+  name: url
+  required: true
+  type: 0
+- defaultvalue: '50'
+  name: max_fetch
+  required: false
+  type: 0
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- additionalinfo: Public token obtained by creating an API token pair on the /config
+    configuration page
+  display: Public API Token
+  hidden: false
+  name: public_api_token
+  required: true
+  type: 4
+- additionalinfo: Private token obtained by creating an API token pair on the /config
+    configuration page
+  display: Private API Token
+  hidden: false
+  name: private_api_token
+  required: true
+  type: 4
+- additionalinfo: Minimum Darktrace score for fetched incidents (0-100)
+  defaultvalue: '0'
+  display: Minimum Score
+  hidden: false
+  name: min_score
+  required: true
+  type: 0
+- additionalinfo: Maximum number of Darktrace model breaches to fetch at a time
+  defaultvalue: '50'
+  display: Maximum Model Breaches per Fetch
+  hidden: false
+  name: max_alerts
+  required: false
+  type: 0
+- additionalinfo: Time to start fetching the first incidents. Default is to begin
+    fetching 1 day ago. Max number of model breaches that will be populated upon first
+    fetch is 20.
+  defaultvalue: 1 day ago
+  display: First fetch time
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+description: This is the Darktrace integration for getting started.
```

Comment:
Please add a more explanatory description saying shortly what the integration does and when you use.

Reply:
Should now be updated